### ISSUE TITLE
To include

### ISSUE THREAD

**Issue by nikosarcevic** [nikosarcevic/HEP-ASTRO-COSMO#11]
A list of TO-INCLUDEs

- [ ] go thru this paper https://arxiv.org/abs/1905.13189
- [ ] check inspire page https://inspirehep.net/experiments?sort=mostrecent&size=25&page=1&q=&classification=Non-experimental
- [ ] comb thru https://ascl.net/home/getwp/958

**Comment by suchitakulkarni** [comment]
@nikosarcevic I don't think we should comb through ascl, should we not just link to them? This will otherwise be a massive effort. ascl interface is great as is

**Comment by nikosarcevic** [comment]
@suchitakulkarni  yeah I should have written "include" ascl, have no clue why I wrote "comb thru".
 will add that later

**Comment by STSpencer** [comment]
Hi!

Thanks for providing this list. I've got some more suggestions that people might find it useful to know about (I don't know what the requirements for are inclusion on the list), all for under ASTRO, though some categories don't exist yet:

Under AGN modelling: 
agnpy https://github.com/cosimoNigro/agnpy

Specific to IACT/gamma-ray analysis:

Fermipy (Very useful for analysis of data from the Fermi Space Telescope) https://github.com/fermiPy/fermipy

gammaALPs (For study of axion-like-particles in gamma-ray data) https://github.com/me-manu/gammaALPs

ctools and gammapy (Very useful for high level Cherenkov Telescope data analysis ) https://github.com/ctools/ctools and https://github.com/gammapy

ctapipe (Useful for low level CTA analysis, though can also be used with data from other IACTs) https://github.com/cta-observatory/ctapipe

Eventdisplay (Similarly for IACT analysis, older but still used and open source) https://github.com/Eventdisplay/Eventdisplay

ShowerModel and pschitt (For simple modelling of air showers and their associated Cherenkov light) https://github.com/JaimeRosado/ShowerModel and https://github.com/vuillaut/pschitt

CTLearn and gammalearn (For deep learning research for IACTs) https://github.com/ctlearn-project/ctlearn and https://gitlab.lapp.in2p3.fr/GammaLearn/GammaLearn

**Comment by nikosarcevic** [comment]
> @suchitakulkarni yeah I should have written "include" ascl, have no clue why I wrote "comb thru".
> will add that later

Hello,
I can add you as a collaborator if you want. But first of all thank you for contributing! This is super useful! Let me know re collaborator status. Otherwise I will be on this later as I really need to finish something for tomorrow.

**Comment by nikosarcevic** [comment]
> Hi!
> 
> Thanks for providing this list. I've got some more suggestions that people might find it useful to know about (I don't know what the requirements for are inclusion on the list), all for under ASTRO, though some categories don't exist yet:
> 
> Under AGN modelling:
> agnpy https://github.com/cosimoNigro/agnpy
> 
> Specific to IACT/gamma-ray analysis:
> 
> Fermipy (Very useful for analysis of data from the Fermi Space Telescope) https://github.com/fermiPy/fermipy
> 
> gammaALPs (For study of axion-like-particles in gamma-ray data) https://github.com/me-manu/gammaALPs
> 
> ctools and gammapy (Very useful for high level Cherenkov Telescope data analysis ) https://github.com/ctools/ctools and https://github.com/gammapy
> 
> ctapipe (Useful for low level CTA analysis, though can also be used with data from other IACTs) https://github.com/cta-observatory/ctapipe
> 
> Eventdisplay (Similarly for IACT analysis, older but still used and open source) https://github.com/Eventdisplay/Eventdisplay
> 
> ShowerModel and pschitt (For simple modelling of air showers and their associated Cherenkov light) https://github.com/JaimeRosado/ShowerModel and https://github.com/vuillaut/pschitt
> 
> CTLearn and gammalearn (For deep learning research for IACTs) https://github.com/ctlearn-project/ctlearn and https://gitlab.lapp.in2p3.fr/GammaLearn/GammaLearn

Hello,
I can add you as a collaborator if you want. But first of all thank you for contributing! This is super useful! Let me know re collaborator status. Otherwise I will be on this later as I really need to finish something for tomorrow.

**Comment by STSpencer** [comment]
Hi!
Great :) I'm happy to collaborate as a gamma-ray person if you'll let me! There's no rush, I'm happy to take part in discussions about what to include and so on, and I'm willing to write a more detailed description of each package if its needed. I wish something like this had been around when I started out!

**Comment by nikosarcevic** [comment]
> Hi!
> Great :) I'm happy to collaborate as a gamma-ray person if you'll let me! There's no rush, I'm happy to take part in discussions about what to include and so on, and I'm willing to write a more detailed description of each package if its needed. I wish something like this had been around when I started out!

absolutely more than welcome! looking forward to collaborating! and thanks in advance - this has to be a community effort as there is no way I can do it myself - I do not have the time nor the knowledge. I will Ladd you right away

**Comment by nikosarcevic** [comment]
*add you

**Comment by suchitakulkarni** [comment]
@nikosarcevic rename FeynFamily to Symbolic manipulation tools (mathematica) and add GroupMath, Sym2Int and Susyno there. These are all symbolic manipulators, not numerical tools. I think so far they all belong to hep than cosmo category.

**Comment by jorgetc16** [comment]
Hi, thanks for including what I sent :) I agree with the Symbolic manipulation section, and I would suggest to move SARAH there too.
For the Astro section we could include https://compose.obspm.fr/ , which generates a table with thermodynamic quantities of EoS applied to Supernovae or Neutron Stars.

**Comment by nikosarcevic** [comment]
> Hi, thanks for including what I sent :) I agree with the Symbolic manipulation section, and I would suggest to move SARAH there too.
> For the Astro section we could include https://compose.obspm.fr/ , which generates a table with thermodynamic quantities of EoS applied to Supernovae or Neutron Stars.

not a problem. can you do a pull request maybe? I am currently very busy (and in a meeting)
